src/test/unit/wrappers/http/test_sync_http_wrappers.py

```diff
@@ -434,3 +436,54 @@ def test_aggregating_response_body():
     for _ in range(10):
         update_event_response(host=None, status_code=200, headers=None, body=big_response_chunk)
     assert len(HttpState.previous_response_body) <= len(big_response_chunk)
+
+
+def test_double_request_size_limit_on_error_status_code(context, monkeypatch):
+    import http.client
+    import lumigo_tracer
+
+    http.client = reload(http.client)
+    lumigo_tracer = reload(lumigo_tracer)
+    lumigo_tracer.wrappers.already_wrapped = False
```

Comment:
why?

Reply:
you can monkeypatch some inner function instead.. It will be much easier to understand...